ci-operator/step-registry/cucushift/installer/rehearse/gcp/upi/xpn/OWNERS

```diff
@@ -0,0 +1 @@
+../OWNERS
```

Comment:
This is a soft symbol link file? OWNERS are located in different folders, and different folders are allowed to define different owners, to avoid the OWNERS list change affect each other different folders, I think it is better to create a real OWNER file.

Reply:
I only use soft-links if I'm in the parent OWNERS, and there're updates done by me in the current folder.